docs/add-ins/addin-tutorial.md

```diff
@@ -141,80 +256,63 @@ Locate the line in the manifest that reads `</DesktopFormFactor>`. Above this li
+1. Add the following element as a child of the `bt:Urls` element:
 
-Finally we'll update our resources. The code above referenced labels, tool-tips, and URLs that we need to define before our manifest will be valid.
-
-1. Add the following as a child of the `bt:Urls` element:
-
-   ```xml
+    ```xml
     <bt:Url id="insertGistPaneUrl" DefaultValue="https://localhost:3000/msg-compose/insert-gist.html"/>
-   ```
-1. Change the `DefaultValue` attribute of the `bt:String` element with `id="groupLabel"` to `Git the Gist`.
+    ```
+1. Within the collection of child elements under the `bt:ShortStrings` element, locate the child element with `id="groupLabel"`. Change the `DefaultValue` attribute of this element to **Git the gist**.
+
+    ```xml
+    <bt:String id="groupLabel" DefaultValue="Git the gist"/>
+    ```
 
-   ```xml
-    <bt:String id="groupLabel" DefaultValue="Git the Gist"/>
-   ```
 1. Add the following elements as children of the `bt:ShortStrings` element.
 
-   ```xml
-    <bt:String id="insertGistLabel" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertGistTitle" DefaultValue="Insert Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert Default Gist">
-      <bt:Override Locale="es-ES" Value="Inserte el Gist predeterminado"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistLabel" DefaultValue="Insert gist"/>
+    <bt:String id="insertGistTitle" DefaultValue="Insert gist"/>
+    <bt:String id="insertDefaultGistLabel" DefaultValue="Insert default gist"/>
+    <bt:String id="insertDefaultGistTitle" DefaultValue="Insert default gist"/>
+    ```
+
 1. Add the following elements as children of the `bt:LongStrings` element.
 
-   ```xml
-    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your Gists and allows you to insert their contents into the current message">
-      <bt:Override Locale="es-ES" Value="Muestra una lista de sus Gists y permite insertar su contenido en el mensaje actual"/>
-    </bt:String>
-    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the Gist you mark as default into the current message">
-      <bt:Override Locale="es-ES" Value="Inserta el contenido de lo Gist que marca como predeterminado en el mensaje actual"/>
-    </bt:String>
-   ```
+    ```xml
+    <bt:String id="insertGistDesc" DefaultValue="Displays a list of your gists and allows you to insert their contents into the current message"/>
+    <bt:String id="insertDefaultGistDesc" DefaultValue="Inserts the contents of the gist you mark as default into the current message"/>
```

Comment:
nit: Maybe "Inserts the content of the..."

Reply:
Will make this change as part of my new PR (coming momentarily).